src/components/Avatar/AvatarGroup.tsx

```diff
@@ -30,8 +31,14 @@ export const AvatarGroup: React.FC<AvatarGroupProps> = React.forwardRef(
     const maxCount: number = maxProps?.count;
 
     const avatarGroupClassNames: string = mergeClasses([
```

Comment:
Once the recommended changes are made in `rtl.scss` there should be no need for the rtl class here.